sdk/avs/Azure.ResourceManager.Avs/src/Customized/CommonClusterProperties.cs

```diff
@@ -46,6 +45,7 @@ public partial class CommonClusterProperties
         private protected IDictionary<string, BinaryData> _serializedAdditionalRawData;
 
         /// <summary> Initializes a new instance of <see cref="CommonClusterProperties"/>. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
since we also do not have a scenario that users would need this type, could we add a `EBNever` attribute to this type as well?